cl/lib/email_backends.py

```diff
@@ -137,6 +138,12 @@ def send_messages(
             # Use base backend connection to send the message
             email.connection = connection
 
+            # Call the random function to determine if we should add or not
+            # a BCC to the message based on the EMAIL_BCC_COPY_RATE set
+            add_bcc = add_bcc_random(settings.EMAIL_BCC_COPY_RATE)
+            if add_bcc == True:
+                email.bcc.append(settings.BCC_EMAIL_ADDRESS)
+
```

Comment:
I think I'd just put all of this into the function, so it's a one-line thing here with no return value. I think semgrep is going to mad at you for doing if xxx == True instead of `is True`, as well. :)

Reply:
Done! I added this logic inside add_bcc_random() and just return the message object with the BCC added or not.